apps/condo/domains/marketplace/components/MarketplaceOnboarding/RequisitesSetup.tsx

```diff
@@ -102,7 +102,7 @@ export const RequisitesSetup: React.FC = () => {
     })
     const updateAction = InvoiceContextApi.useUpdate({ status: INVOICE_CONTEXT_STATUS_INPROGRESS })
 
-    const [noTaxOption]: SelectProps['options'] = [{ label: NoTax, key: NoTax, value: null }]
+    const [noTaxOption]: SelectProps['options'] = [{ label: NoTax, key: NoTax, value: '' }]
```

Comment:
You can do it little bit simple with typescript trick
```suggestion
    const noTaxOption: SelectProps['options'][number] = { label: NoTax, key: NoTax, value: '' }
```

Reply:
With `number` shorthand you can get access to array item type, so you don't need to create array and then unpack it to a single value

Reply:
Refactored